fastestimator/backend/get_lr.py

```diff
@@ -43,15 +43,9 @@ def get_lr(model: Union[tf.keras.Model, torch.nn.Module]) -> float:
         ValueError: If `model` is an unacceptable data type.
     """
     if isinstance(model, tf.keras.Model):
-        if hasattr(model, 'current_optimizer'):
-            lr = tf.keras.backend.get_value(model.current_optimizer.lr)
-        else:
-            lr = tf.keras.backend.get_value(model.optimizer.lr)
+        lr = tf.keras.backend.get_value(model.current_optimizer.lr)
```

Comment:
no. this is there to allow to work with non fe.build models

Reply:
I think the function is intended for models after fe.build. that's why we have the assertion now.


it won't work on non-fe.build model because pytorch module does not have `.optimizer` field by default.